Projects/hCoreUI/Sources/hForm/hRow.swift

```diff
@@ -105,6 +112,15 @@ public struct StandaloneChevronAccessory: View {
     }
 }
 
+public struct StandaloneChevronNewAccessory: View {
```

Comment:
Merge this with StandaloneChevronAccessory
and add environment hUseNewDesign property to decide if you want to use it.